contrib/builder/rpm/amd64/amazonlinux-latest/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/amd64/generate.sh"!
+#
+
+FROM amazonlinux:latest
+
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static  libselinux-devel libtool-ltdl-devel pkgconfig selinux-policy selinux-policy-devel sqlite-devel  tar git cmake vim-common
```

Comment:
This is autogenerated. I could fix the script but that doesn't seem within scope for this pull request, IMO.